### DISCUSSION TITLE
Github Graduation Custom Trading Card Shipment

### DISCUSSION THREAD

**Discussion by srtcdncr** [community/community#20381] [GitHub Education] [answered]
Hello, my PR was accepted to the Github Graduation repo, and I received an email on May 26th that a trading card will arrive at my house. But today is July 12th and I have no information about the shipment. Where can I contact about this?

**Comment by ashr123** [comment]
me Too: education/GitHubGraduation-2022#958

**Comment by Nishchal-007** [comment]
+1

**Comment by samus-aran** [comment, marked as answer]
Hi y'all, just to identify for graduates that have not yet received their Graduation Trading Card shipment you have a chance to fill out a [form for a re-print](https://github.com/education/GitHubGraduation-2022#sep-05-2022). But you only have until **Sept 30th**. 📆
